.github/workflows/canary.yaml

```diff
@@ -42,14 +42,14 @@ jobs:
         if: steps.cache.outputs.cache-hit != 'true'
         run: |
           npm install --ignore-scripts
-          npx lerna bootstrap --no-ci
+          npx lerna bootstrap --no-ci --hoist --nohoist='zone.js'
```

Comment:
do you know why `npx bootstrap` is used here instead of `npm run bootstrap`?
It can make repeating the `--hoist --nohoist='zone.js'` in each workflow redundant.
In addition, it guarantees that the lerna version in use is the one from `package.json` and not the latest which is what will be used automatically by `npx` I think. This will make the builds more stable.

Reply:
npx uses the locally installed version if it exists. Without npx, you need to install lerna globally

Reply:
I suppose In the CI lerna is not installed globally, so it will fallback to latest version instead of the one from `package.json`, right?

Reply:
Will it work to use `npm run bootstrap` which will run `"bootstrap": "lerna bootstrap --hoist --nohoist='zone.js'",` with the lerna installed in node_modules (stable version from package.json)?

Reply:
If you run in a project directory, it looks in the local node_modules before going to the internet to pull a version.

Reply:
Cool, learned something new :)